docs/guides/qiskit-code-assistant.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Qiskit Code Assistant
+description: Learn how to use Qiskit Code Assistant, a generative AI code assistant.
+---
+
+# Qiskit Code Assistant
+
+The Qiskit Code Assistant aims to make quantum computing more accessible to new Qiskit adopters, and improve the coding experience for current users. It is a generative AI code assistant powered by [watsonx](https://www.ibm.com/products/watsonx-ai). It is trained using millions of text tokens from Qiskit SDK v1.x, years of Qiskit code examples and IBM Quantum&trade; features. The Qiskit Code Assistant can help your quantum development workflow by offering LLM-generated suggestions based on [IBM Granite 8B Code,](https://www.ibm.com/products/watsonx-ai/foundation-models) which incorporate the latest features and functionalities from IBM&reg;.
+
+<Admonition type="note" title="Notes">
+    - This is an experimental feature available only to IBM Quantum Premium Plan users.
+    - The Qiskit Code Assistant is in preview release status and is subject to change.
+    - If you have feedback or want to contact the developer team, use the [Qiskit Slack Workspace channel](https://qiskit.enterprise.slack.com/archives/C07LYA6PL83) or the related public GitHub repositories.
+</Admonition>
+
+<video title="A user uses the Qiskit Code Assistant in JupyterLab. The user has a comment in the code to get the available backends for their account in IBM Quantum, the Qiskit Code Assistant gives a code suggestion to get the available backends. The user executes the code, gets the available backends in IBM Quantum Platform for the current account. The user ask to the Qiskit Code Assistant to generate the code to get the T1, T2 and other properties for the qubit 1 in the device ibm_brisbane. The assistant gives a code suggestion. The user runs the suggested code and get the desired output." className="max-w-auto h-auto" controls>
+  <source src="/videos/guides/qiskit-code-assistant/demo-JupyterLab-t1t2-granite-8b-qiskit-short.mp4" type="video/mp4"></source>
+</video>
+
+## Features
+
+* Accelerates Qiskit code generation by leveraging generative AI based on the `granite-8b-qiskit` model.
+* Use abstract and specific prompts to generate recommendations.
```

Comment:
The conjugation is inconsistent here. Ditto rest of list

Reply:
yeahhhh - it's ugly because they're not all really the same type of activity.  Let me see if I can fix.